share/picongpu/examples/FoilLCT/bin/plot_charge_density.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+#
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017 PIConGPU contributors
+Authors: Axel Huebl
+License: GPLv3+
```

Comment:
Should we add the full license header like in all other tools?

Reply:
https://github.com/ComputationalRadiationPhysics/picongpu/pull/2259#issuecomment-330198911